code/modules/research/techweb/all_nodes.dm

```diff
@@ -617,7 +617,7 @@
 	display_name = "Night Vision Technology"
 	description = "Allows seeing in the dark without actual light!"
 	prereq_ids = list("integrated_HUDs", "adv_engi", "emp_adv")
-	design_ids = list("health_hud_night", "security_hud_night", "diagnostic_hud_night", "night_visision_goggles", "nvgmesons")
+	design_ids = list("health_hud_night", "security_hud_night", "diagnostic_hud_night", "diagnostic_hud_weld", "night_visision_goggles", "nvgmesons")
```

Comment:
Night vision tech seems like an odd place for it.

Also, put a comment at the end that there is a monkestation edit.